code-challenges.js

```diff
@@ -59,3 +104,16 @@ const number2 = 100
 // Set two:
 const number3 = 27
 const number4 = 24
+
+
+const bigDifference = (digit1, digit2) => {
+    if (digit1 < digit2) {
+    return digit2 - digit1
+    }else if (digit1 > digit2) {
+        return digit1 - digit2
+    }else if (digit1 === digit2){
+        return `${digit1} and ${digit2} are equal`
+    }
+}
+console.log (bigDifference(number1, number2)) // 42
+console.log (bigDifference(number3, number4)) // 3
```

Comment:
Will, you rocked these challenges! Fantastic job with pseudocode - and I love your variable names. You can definitely stretch yourself by trying out edge cases or using other input not explicitly provided!  For future assessments, please be sure to uncomment your code so it is clear and easy to read.